.gitignore

```diff
@@ -49,7 +49,7 @@
 /var/tmp/*
 !/var/tmp/.gitkeep
 
-/app/config/local*.php
```

Comment:
Could we keep the wild card there? I often copy the local config as a backup before I test some PR that modifies it so I could compare the changes and also have a working config file to get back to.

Reply:
I'm also thinking if `config_local.php` is the right name. We mention `local.php` in all docs and forum threads. Does it have any advantage to name it? Perhaps even `config/local.php` would be enough? So the only thing that would change would be to move it from the app dir to the root dir.

Reply:
I agree with @escopecz there is no need to rename this file, there is a lot of material written using `local.php` so changing the name will lead to a lot of support work compared to changing the path

Reply:
I renamed the file to be in line with the other files (`parameters_local.php`, `paths_local.php`, ...), but I can agree with keeping the name.
I'll adjust the PR